aten/src/ATen/CPUApplyUtils.h

```diff
@@ -151,6 +151,7 @@ struct strided_tensor_iter_fixed {
   int64_t strides_[N] = {0};
 
   strided_tensor_iter_fixed(strided_tensor_iter_fixed const&) = delete;
+  operator bool() const { return CanOverflow; };
```

Comment:
I found this a bit unexpected. Why does overflow have anything to do with whether or not an iterator converts to bool? Why should "can overflow" be privileged in this way?

Reply:
Agree, changing to:
bool Overflow() const { return CanOverflow; };